lib/tool_belt/release_environment.rb

```diff
@@ -24,23 +24,9 @@ def setup(args = {})
           Dir.chdir(name.to_s) do
             @systools.execute("git remote set-url origin #{add_username(repo[:repo], github_username)}") if github_username
 
-            if repo[:branch]
-              if branch_exists?("origin/#{repo[:branch]}") &&
-                @systools.execute("git checkout origin/#{repo[:branch]}")
-              elsif tag_exists?(repo[:branch]) &&
-                @systools.execute("git checkout #{repo[:branch]}")
-              else
```

Comment:
We still want this functionality or else when you setup environments it won't setup the branch properly based on the config.

Reply:
The only part that conflicts with your new commands is the line below this that attempts to create a branch if it does not exist.